content/docs/04.5-devx/03-cluster-groups/35-ingress-cluster-group.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Set Up Ingress"
+metaTitle: "Set Up Ingress for Cluster Groups"
+metaDescription: "Learn how to set up a Kubernetes Ingress for a Palette Cluster Group"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Overview
+
+Cluster Groups may have a cluster endpoint type of either Load Balancer or Ingress. The cluster endpoint type determines how Palette Virtual Clusters deployed in a Cluster Group are exposed. You specify the cluster endpoint in Cluster Group Settings.
+
+Using **Ingress** as the cluster endpoint type is a more cost effective way to access your Kubernetes workloads than using type **Load Balancer**, which requires a new cloud load balancer to be provisioned for each virtual cluster.
+
+When you enable **Ingress** as the endpoint for a Cluster Group, you must deploy an NGINX Ingress Controller add-on profile on each host cluster in the Cluster Group. The NGINX Ingress Controller provides the necessary routing functionality for external traffic to reach the Kubernetes API server of each virtual cluster, as well as any apps each virtual cluster contains.
+
+# Prerequisite
+
+- At least one infrastructure or cloud-based cluster you’ve created at the tenant level.
+
+# Set Up Ingress
+
+1. Log in to Palette as **Tenant Admin**.
+<br />
+2. Identify each host cluster that requires the addition of an NGINX Ingress Controller profile.
+
+    This can be:
+
+    - All the host clusters in an existing Cluster Group, <br />
+        or
+    
+    - Existing host clusters that you will add to a new Cluster Group you'll create later. <br /><br />
+
+3. Add ```ingress-nginx-host-cluster``` profile to each host cluster. <br />
+    a. From the main menu, choose **Clusters** and select a cluster.<br />
+    b. In the **Profile** tab, click **Add add-on profile (+)** and select ```ingress-nginx-host-cluster```. <br />
+    c. Confirm and save your changes.
+<br />
+4. For each host cluster with ingress deployed, use webkubectl and invoke the ```kubectl -n nginx get service nginx-ingress-controller``` command to identify the External-IP of the ```ingress-nginx-host-cluster``` LoadBalancer service. 
```

Comment:
"LoadBalancer service." should be "LoadBalancer Service."

Services are a 1st class Kubernetes resource and are typically capitalized - like Pod, Deployment, ConfigMap, etc.

Reply:
Done